lib/ex_check/config.ex

```diff
@@ -1,8 +1,6 @@
 defmodule ExCheck.Config do
   @moduledoc false
 
-  alias __MODULE__.{Generator, Loader}
```

Comment:
I agree that using full module names has its benefits. On the other hand, there are more instances of `__MODULE__` that you haven't changed so that feels a little bit inconsistent. Up to you to decide :)

Reply:
I can change the others in a parallel PR. Will open it when I do the rest of this.

Reply:
Parallel PR opened in #22